nifi-extension-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service-api/pom.xml

```diff
@@ -36,9 +36,8 @@
             <artifactId>nifi-proxy-configuration-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
```

Comment:
The `ObjectMapper` is referenced in `IndexOperationResponse`, so this dependency should be retained, but the scope should be `provided`.

Reply:
Good catch!

Reverted the removal and changed the `scope` to `provided`